packages/create/src/cli.ts

```diff
@@ -33,9 +35,23 @@ program
       selectedTemplate = template;
     }
 
-    console.log('🚧 create-sku is not yet implemented');
-    console.log(`Project name: ${projectName}`);
-    console.log(`Template: ${selectedTemplate}`);
+    if (!selectedTemplate) {
+      console.log('❌ No template selected. Exiting.');
+      process.exit(1);
+    }
+
```

Comment:
I think we need a warning about the experimental nature of the Vite template here.